src/server/server.rs

```diff
@@ -59,7 +59,7 @@ impl<T: RaftStoreRouter, S: StoreAddrResolver + 'static> Server<T, S> {
         region_split_size: usize,
         storage: Storage,
         raft_router: T,
-        snapshot_status_sender: Sender<SnapshotStatusMsg>,
+        significant_msg_sender: Sender<SignificantMsg>,
```

Comment:
can we move the sender to the RaftRouter? I think we can only use raft router to communicate with raft store.

So the RaftRouter trait may add:

```
fn report_unreachable()
fn report_snapshot_status()
```

Reply:
Tried, but got a lot of unrelated changes, maybe in another pr.

Reply:
Got it. I think it is worth to do it later.